doc/sailfish-webview-webview.qdoc

```diff
@@ -26,11 +26,9 @@
   ApplicationWindow {
       id: root
       initialPage: Component {
-          Page {
-              id: page
+          WebViewPage {
```

Comment:
I've used this code as a base for other examples, so will aim to amend those in some other open PRs too.

Reply:
Thanks